src/types/field/emailField.ts

```diff
@@ -13,4 +13,6 @@ export interface IEmailField extends IField {
   isVerifiable: boolean
 }
 
-export interface IEmailFieldSchema extends IEmailField, IFieldSchema {}
+export interface IEmailFieldSchema extends IEmailField, IFieldSchema {
+  isVerifiable: boolean
+}
```

Comment:
@karrui why was this necessary? isn't this already in `IEmailField`

Reply:
Sigh, pain point with discriminators. `IFieldSchema` once again overrode `isVerifiable` back to an optional field, but we want it to be correctly required, so we need to override it again.